modules/kn-service-offline-create.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="creating-an-offline-service_{context}"]
+= Creating a service using offline mode
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have installed `kn` CLI.
+
+.Procedure
+
+. In the offline mode, create a local Knative service descriptor file:
++
+[source,terminal]
+----
+$ kn service create event-display \
+    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest \
+    --target ./ \
+    --namespace test
+----
++
+[source,terminal]
+----
+Service 'event-display' created in namespace 'test'.
+----
++
+* The `--target ./` flag enables the offline mode and specifies `./` as the directory for storing the new directory tree.
++
+If you do not specify an existing directory, but use a filename, such as `--target my-service.yaml`, then no directory tree is created. Instead, only the service descriptor file `my-service.yaml` is created in the current directory.
++
+The filename can have the `.yaml`, `.yml`, or `.json` extension. Choosing `.json` creates the service descriptor file in the JSON format.
++
+* The `--namespace test` option places the new service in the `test` namespace.
++
+If you do not use `--namespace`, and you are logged in to an OpenShift cluster, the descriptor file is created in the current namespace. Otherwise, the descriptor file is created in the `default` namespace.
+
+. Examine the created directory structure:
++
+[source,terminal]
+----
+$ tree ./
+./
```

Comment:
Please split the top command from the output, per the [OCP guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output), with `.Example output` before the output code block.